improver/cube_combiner.py

```diff
@@ -54,13 +62,16 @@ class CubeCombiner(BasePlugin):
         "mean": np.add,
     }  # mean is calculated in two steps: sum and normalise
 
-    def __init__(self, operation, warnings_on=False):
+    def __init__(self, operation, broadcast_to_coords=None, warnings_on=False):
         """
         Create a CubeCombiner plugin
 
         Args:
             operation (str):
                 Operation (+, - etc) to apply to the incoming cubes.
+            broadcast_to_coords (list):
+                Specifies a list of coord names that exist only on the first cube that
+                the other cube(s) need(s) broadcasting to prior to the combine.
```

Comment:
This relates to the properties of the input cubes - it might be better as a `process` argument.

Reply:
So does "new_name". I was just trying to be consistent with what was already present. do you want me to move several? If so, which?

Reply:
I think we're at cross-purposes here - "new_name" is already a `process` argument.  I would suggest `broadcast_to_coords` (or `broadcast_across_thresholds`) is conceptually similar and therefore also belongs better in `process`.

Reply:
I've moved broadcat_to_coords from init to process